.github/workflows/build_and_unit_test.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and unit test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Booooo :-1:

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

@vpodzime Which one do you want?

Reply:
:disappointed: 

```
Ubuntu 20.04 | ubuntu-20.04
Ubuntu 18.04 | ubuntu-latest or ubuntu-18.04
```
??? Let's at least use the latest Ubuntu and not *ubuntu-latest*.

Reply:
Changed to Ubuntu 20.04